cve_bin_tool/version_scanner.py

```diff
@@ -125,6 +125,9 @@ def is_executable(self, filename: str) -> tuple[bool, str | None]:
                 ("LSB " not in output)
                 and ("LSB shared" not in output)
                 and ("LSB executable" not in output)
+                and ("MSB " not in output)
+                and ("MSB shared" not in output)
+                and ("MSB executable" not in output)
```

Comment:
```suggestion
```
Wouldn't line 128 cover the other conditions on line 129 and 130?

Reply:
Indeed, it was basically a copy/paste of LSB which was already redundant. I updated the PR as requested

Reply:
That makes sense. I'll file an issue to get that resolved.